src/components/TextAndMediaItem/Media/MediaImage.tsx

```diff
@@ -25,7 +25,7 @@ type MediaImageProps = {
 export const MediaImage = ({ data, alt }: MediaImageProps) => {
   return (
     <StyledMediaContainer>
-      <StyledImage src={data} alt={alt} />
+      <StyledImage src={data} alt={alt} loading="lazy" />
```

Comment:
```suggestion
      <StyledImage loading="lazy" src={data} alt={alt} />
```

There's a bug in Firefox - [ref](https://bugzilla.mozilla.org/show_bug.cgi?id=1647077) and it will be safer to put `loading` attribute before `src`.

Can we also add `loading` to other images? I mean `FeaturedCase` & `TechnologyListSection`

Reply:
oo, that's good to know, thanks! 

I added `loading` to the `TechnologyListSection`, like you suggested, but I am not is a good idea to add it also to the `FeaturedCase`. There we have the image that is also LCP on our website and I read that you shouldn't add lazy loading to the LCP.